testing/run-scripts/run_cloud.sh

```diff
@@ -52,6 +52,12 @@ then
     usage
 fi
 
+if [ "$WIPE" = true ] && [ "$UPDATE" = false ]
+then
+    echo "-u must be used when -w is used"
```

Comment:
Do we want to explicitly require these two flags to always be together? If so, maybe just combine into one flag?